client/src/components/GameInfoCarousel/GameInfoCarousel.jsx

```diff
@@ -62,29 +66,35 @@ const GameInfoCarousel = () => {
       setMainImage,
```

Comment:
This seems to be part of react hooks which I'm still not very familiar with, overall it's awesome that you've picked that up and are able to use it! Impressive!